doc/source/whatsnew/v0.20.0.txt

```diff
@@ -382,5 +382,4 @@ Bug Fixes
 
 - Bug in ``Series`` constructor when both ``copy=True`` and ``dtype`` arguments are provided (:issue:`15125`)
 - Bug in ``pd.read_csv()`` for the C engine where ``usecols`` were being indexed incorrectly with ``parse_dates`` (:issue:`14792`)
-
-- Bug in ``Series.dt.round`` inconsistent behaviour on NAT's with different arguments (:issue:`14940`)
+- Not boolean Series returned by comparison methods (e.g., ``lt``, ``gt``, ...) against a constant for an empty DataFrame (:issue:`15077`)
```

Comment:
say

Incorrect dtyped Series was returned by comparions methods.........against a constant for an empty DataFrame

put Series/DataFrame with double-backticks.